src/long-press-directive.ts

```diff
@@ -1,63 +1,76 @@
-import { directive, PropertyPart } from "lit-html";
+import { directive, PropertyPart } from 'lit-html';
 // See https://github.com/home-assistant/home-assistant-polymer/pull/2457
 // on how to undo mwc -> paper migration
-// import "@material/mwc-ripple";
-// import "@polymer/paper-ripple";
+// import '@material/mwc-ripple';
 
-const isTouch =
-  "ontouchstart" in window ||
-  navigator.maxTouchPoints > 0 ||
-  navigator.msMaxTouchPoints > 0;
+const isTouch = 'ontouchstart' in window
+  || navigator.maxTouchPoints > 0
+  || navigator.msMaxTouchPoints > 0;
 
-interface LongPressInterface extends HTMLElement {
+interface LongPress extends HTMLElement {
```

Comment:
I don't personally like that the interface and class share the same name, hence why I purposely changed this

Reply:
Okay 👍